extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussions.i18n.php

```diff
@@ -11,6 +11,7 @@
 	'embeddable-discussions-zero' => 'No posts yet.',
 	'embeddable-discussions-zero-detail' => 'Get the discussion started, create the first post now!',
 	'embeddable-discussions-forum-name' => 'In $1',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "In $1" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "embeddable-discussions-forum-name" 12 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)